source/plugins/go/src/go.mod

```diff
@@ -1,6 +1,8 @@
 module Docker-Provider/source/plugins/go/src
 
-go 1.19
+go 1.21
```

Comment:
same here, what's reason for using go version 1.21 instead of 1.22.5?

Reply:
replied above